app/components/viral/form/prefixed/text_input_component.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Viral
+  module Form
+    module Prefixed
+      # Viral form input with a prefix that can contain text or svg providing additional context to the input
+      class TextInputComponent < Viral::Component
+        attr_reader :form, :name, :value, :pattern, :placeholder, :required
+
+        renders_one :prefix
+
+        # rubocop:disable Metrics/ParameterLists
```

Comment:
**nitpick (blocking):** Can you move your rubocop disable onto the same line as the `def` that way you don't have to call `enable` later on.

Reply:
Updated in a3ee2f2927e33178e3afd0bdd11daddadd5d3ccc